soc/arm/microchip_mec/common/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if (NOT DEFINED SOC_SERIES_MEC1701X)
```

Comment:
What is this line fore?

Reply:
Prevent this files from being used with MEC170x since it has an old, old CMSIS header that is incompatible with newer parts.  Updating MEC170x is in the queue but has low priority ;<(

Reply:
What is your preference for splitting the large commit?  There are > 30 headers one for each kind of peripheral.  When I split I can add git tags/labels.  Which tag should I use: annotated or lightweight?